Gaijinizer patch/Map038.txt

```diff
@@ -114,19 +116,18 @@ Reina. Because that niece has been transferred
 レイナは、俺の魔力や、お前が目覚めない原因が
 すべてＭ.Ｏ.Ｗにあると結論を出した。
 > CONTEXT: Map038/events/4/pages/0/66/Dialogue/yuuma
-It was about two years before the war situation
-started to smolder. Reina is my magic and why you
-don't wake up. It concluded that all were in
-M.O.W.#MTLed
+It was about two years before this war started to
+get hot. Reina came to the conclusion that the
+cause of my magic, and your coma, was all M.O.W.#Ron
```

Comment:
Flow:
>_**It was about two years before this war started to
 reach its boiling point. Reina came to the conclusion that
the cause of my magic and your coma was all M.O.W's fault.**_